rs-commerce/src/views/registration/components/RegistrationForm/components/AddressRegistration/components/AddressComponents/components/AddresTitleComponent/AddresTitleComponent.tsx

```diff
@@ -0,0 +1,25 @@
+import classes from './style.module.css';
+
+export type AddressType = {
+  typeComponent: 'billing' | 'shipping';
+};
+
+function AddressTitleComponent(props: AddressType) {
+  const { typeComponent } = props;
+  const typeComponentUpper: string = `${typeComponent[0]?.toUpperCase()}${typeComponent?.slice(1)}`;
+  return (
+    <div className={classes['address__title-block']}>
+      <h3 className={classes['address__subtitle']}>{`${typeComponentUpper} address`}</h3>
+      <label className={classes['input__label']} htmlFor={`set${typeComponentUpper}DefaultAddress`}>
+        <input
+          className={classes['input_checkbox']}
+          type="checkbox"
+          id={`set${typeComponentUpper}DefaultAddress`}
+        />
+        <p className={classes['checkbox__content']}>Set as a default address</p>
+      </label>
```

Comment:
You can use separate Checkbox component here the same like in `AddressRegistration` component. Difference will be in props